client/src/partials/home/home.js

```diff
@@ -45,135 +45,18 @@ function HomeController(Currencies, Rates, Session, System, $interval, $translat
     .then(function () {
       vm.currencies.forEach(function (currency) {
         currency.rate = Rates.getCurrentRate(currency.id);
+        currency.formattedDate = new Moment(currency.date).format('LL');
       });
+
+      // @TODO Method for selecting primary exchange
+      vm.primaryExchange = vm.currencies[0];
```

Comment:
This could always be the first element if the list is length 1.  Otherwise, it might be a setting on the settings page.